Xbox One build is flaky

It appears to occasionally fail to download zlib (checksum mismatch), which means sdl2_image fails to build, causing the build to fail.

Example: https://github.com/diasurgical/devilutionX/actions/runs/3719831829/jobs/6308932249

```
  -- verifying file...
         file='D:/a/devilutionX/devilutionX/build/_deps/zlib-subbuild/zlib-populate-prefix/src/zlib-1.2.13.tar.gz'
  -- MD5 hash of
      D:/a/devilutionX/devilutionX/build/_deps/zlib-subbuild/zlib-populate-prefix/src/zlib-1.2.13.tar.gz
    does not match expected value
      expected: '9b8aa094c4e5765dabf4da391f00d15c'
        actual: '63ef3ce8f23f8f4f26081a96e09755e4'
  -- Hash mismatch, removing...
  CMake Error at zlib-subbuild/zlib-populate-prefix/src/zlib-populate-stamp/download-zlib-populate.cmake:170 (message):
    Each download failed!
```

```
LINK : fatal error LNK1181: cannot open input file 'sdl_image.lib' [D:\a\devilutionX\devilutionX\uwp-project\devilutionx.vcxproj]
```

--- a/3rdParty/zlib/CMakeLists.txt
+++ b/3rdParty/zlib/CMakeLists.txt
@@ -6,6 +6,7 @@ include(FetchContent)
 FetchContent_Declare(zlib
     URL https://www.zlib.net/zlib-1.2.13.tar.gz
         https://www.zlib.net/fossils/zlib-1.2.13.tar.gz
+        https://github.com/madler/zlib/releases/download/v1.2.13/zlib-1.2.13.tar.gz
     URL_HASH MD5=9b8aa094c4e5765dabf4da391f00d15c
 )
 FetchContent_MakeAvailableExcludeFromAll(zlib)

